getting-started/index.html

```diff
@@ -146,7 +146,7 @@ <h2>Taking it further</h2>
 <p>To further develop your knowledge and encounter other topics we recommend the following.</p>
 <ul>
 <li>
-<p> <a class="highlightLink"  href="/International/i18n-drafts/techniques/authoring-html.en">Internationalization techniques: Authoring HTML &amp; CSS</a><br>Resources on the W3C Internationalization Activity site and elsewhere that help you author HTML and CSS for internationalization.</p>
+<p> <a class="highlightLink"  href="/International/i18n-drafts/techniques/authoring-html.en">Internationalization techniques: Authoring Web pages</a><br>Resources on the W3C Internationalization Activity site and elsewhere that help you author HTML and CSS for internationalization.</p>
```

Comment:
Authoring Web pages should either be:
Authoring Web Pages, or
Authoring web pages
(probably the latter)

Reply:
I copied it from https://www.w3.org/International/techniques/authoring-html . Should we change the capitalization there too?

Reply:
Yes, i think so.

Reply:
Thanks. Should be fixed now.